code/game/gamemodes/devil/devil_game_mode.dm

```diff
@@ -62,4 +62,27 @@
 	sleep(100)
 	add_devil_objectives(devil, objective_count) //This has to be in a separate loop, as we need devil names to be generated before we give objectives in devil agent.
 	devil.announceDevilLaws()
-	devil.announce_objectives()
\ No newline at end of file
+	devil.announce_objectives()
+
+/proc/get_devil_datum(mob/living/M)
```

Comment:
I would rather you use the mind proc(has_antag_datum) than adding a new one just for this. This is a pain to maintain later on.